migrations/postgres/001_create_users.up.sql

```diff
@@ -0,0 +1,31 @@
+-- Users table
+CREATE TABLE users (
+    user_id SERIAL PRIMARY KEY,
+    phone_number VARCHAR(20) UNIQUE,
+    email VARCHAR(255) UNIQUE,
+    first_name VARCHAR(100),
+    last_name VARCHAR(100),
+    phone_verified BOOLEAN DEFAULT false,
+    phone_verification_code VARCHAR(6),
```

Comment:
why is phone_verification_code required?

Reply:
Redis will be used for temporary storage of the verification codes